src/main/webapp/app/exam/manage/exam-management.component.ts

```diff
@@ -53,9 +50,7 @@ export class ExamManagementComponent implements OnInit, OnDestroy {
         this.courseService.find(Number(this.route.snapshot.paramMap.get('courseId'))).subscribe(
             (res: HttpResponse<Course>) => {
                 this.course = res.body!;
-                this.isAtLeastInstructor = this.accountService.isAtLeastInstructorInCourse(this.course);
-                this.isAtLeastEditor = this.accountService.isAtLeastEditorInCourse(this.course);
-                this.isAtLeastTutor = this.accountService.isAtLeastTutorInCourse(this.course);
+                this.accountService.setAccessRightsForCourse(this.course);
```

Comment:
In the future this method call should move into `courseService` when the response is handled.

Reply:
Fixed in https://github.com/ls1intum/Artemis/pull/3742/commits/a6459f036b37b25a1336990f5f73882c2cf108c5